src/cloud/internal/cloudconfiguration.cpp

```diff
@@ -106,6 +106,12 @@ QUrl CloudConfiguration::signUpUrl() const
     return QUrl("https://musescore.com/oauth/authorize-new");
 }
 
+QUrl CloudConfiguration::signInSuccessUrl() const
+{
+    // TODO
+    return QUrl("https://musescore.com");
```

Comment:
FYI, the homepage URL redirects to https://musescore.com/dashboard for logged-in users.